### PR TITLE
Add Python3 support using Six.

### PR DESCRIPTION
**Pull request by xals**

### CODE REVIEW

File: ocf/ra.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 from lxml import etree
 from ocf.util import cached_property
 
+import six
```
Comment (bootc) [inline ocf/ra.py:27]: Please add an extra newline after the import statement.

File: ocf/ra.py [hunk 2]
```diff
@@ -606,16 +607,16 @@ def meta_data(self):
...
 
         actions = etree.SubElement(xra, 'actions')
-        for action in self._ACTIONS.itervalues():
+        for action in six.itervalues(self._ACTIONS):
             action.append_xml(actions)
 
         print(etree.tostring(
             xra, pretty_print=True, xml_declaration=True, encoding='utf-8',
-            doctype='<!DOCTYPE resource-agent SYSTEM "ra-api-1.dtd">'))
+            doctype='<!DOCTYPE resource-agent SYSTEM "ra-api-1.dtd">').decode('UTF-8'))
```
Comment (bootc) [inline ocf/ra.py]: Please break this long line so that it does not exceed 79 characters.

**Comment by bootc** [comment]
Hi @xals, thanks very much for your contribution! It looks good to me barring two very minor tweaks that I'd like to see to ensure PEP8 compliance. I tested this using `flake8` which also shows up an error on my part.

I'd be happy to accept an updated pull, else I can fix this up for you myself if you'd rather - just let me know.

Thanks again,
Chris

**Comment by xals** [comment]
Applied both fixes.

**Comment by xals** [comment]
@bootc Thanks to you, I was writing exactly the same thing before I find you library!

**Pull request merged**